File: MMORPG_2/Assets/Script/Data/LocalData/CreateLua/ChapterLevelEntity.lua
ChapterLevelEntity = { Id = 0, Chapter = 0, Level = 0, Name = "", WorldMapId = 0, Star2 = 0, Star3 = 0, IsBoos = 0, FolderName = "", IconName = "", RecommendFight = 0, MaxCount = 0, NeedVitality = 0, FirstThreeStar = 0, Desc = "" }

--这句是重定义元表的索引，就是说有了这句，这个才是一个类
ChapterLevelEntity.__index = ChapterLevelEntity;

function ChapterLevelEntity.New(Id, Chapter, Level, Name, WorldMapId, Star2, Star3, IsBoos, FolderName, IconName, RecommendFight, MaxCount, NeedVitality, FirstThreeStar, Desc)
    local self = { }; --初始化self
    setmetatable(self, ChapterLevelEntity); --将self的元表设定为Class

    self.Id = Id;
    self.Chapter = Chapter;
    self.Level = Level;
    self.Name = Name;
    self.WorldMapId = WorldMapId;
    self.Star2 = Star2;
    self.Star3 = Star3;
    self.IsBoos = IsBoos;
    self.FolderName = FolderName;
    self.IconName = IconName;
    self.RecommendFight = RecommendFight;
    self.MaxCount = MaxCount;
    self.NeedVitality = NeedVitality;
    self.FirstThreeStar = FirstThreeStar;
    self.Desc = Desc;

    return self;
end

File: GameServer/GameServerApp/GameServerApp/Data/LocalData/Create/ChapterLevelEntity.cs

//===================================================
//作    者：
//创建时间：2019-05-18 11:12:46
//备    注：此代码为工具生成 请勿手工修改
//===================================================
using System.Collections;

/// <summary>
/// ChapterLevel实体
/// </summary>
public partial class ChapterLevelEntity : AbstractEntity
{
    /// <summary>
    /// 所属章节
    /// </summary>
    public int Chapter { get; set; }

    /// <summary>
    /// 章节等级
    /// </summary>
    public int Level { get; set; }

    /// <summary>
    /// 图标名称
    /// </summary>
    public string Name { get; set; }

    /// <summary>
    /// 世界地图Id
    /// </summary>
    public int WorldMapId { get; set; }

    /// <summary>
    /// 2星通关时间
    /// </summary>
    public int Star2 { get; set; }

    /// <summary>
    /// 3星通关时间
    /// </summary>
    public int Star3 { get; set; }

    /// <summary>
    /// 是否Boss
    /// </summary>
    public int IsBoos { get; set; }

    /// <summary>
    /// 背景所在文件夹名
    /// </summary>
    public string FolderName { get; set; }

    /// <summary>
    /// 图片名
    /// </summary>
    public string IconName { get; set; }

    /// <summary>
    /// 推荐战力
    /// </summary>
    public int RecommendFight { get; set; }

    /// <summary>
    /// 最大挑战次数
    /// </summary>
    public int MaxCount { get; set; }

    /// <summary>
    /// 所需活力
    /// </summary>
    public int NeedVitality { get; set; }

    /// <summary>
    /// 首次三星奖励绑金数
    /// </summary>
    public int FirstThreeStar { get; set; }

    /// <summary>
    /// 图标名称
    /// </summary>
    public string Desc { get; set; }

}
